### PR TITLE
prettier config

### PR DESCRIPTION
**Pull request by Caleb335**
I've added the prettier dependency to this project, and I tested the npm script, everything works fine, `npm run format` does the formatting for you.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +19,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "format": "prettier --write \"src/**/*.{js,jsx,css,html,scss}\""
```
Comment (madhuni) [inline package.json:23]: We can add additional scripts to check the format. e.g.:
`"format:check": "prettier -c  \"src/**/*.{js,jsx,css,html,scss}\"`

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "singleQuote": true,
+    "tabWidth": 4,
+    "semi": false,
```
Comment (madhuni) [inline .prettierrc:4]: I think we should have semicolons and we don't need to remove them.

Comment (Caleb335) [reply .prettierrc:4]: Okay, semicolon it is.

File: .prettierrc [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+    "singleQuote": true,
+    "tabWidth": 4,
```
[the remark this answers is not in the document]
Comment (Caleb335) [reply .prettierrc:3]: @madhuni we can use editorconfig to settle this, maybe you should create an issue based in that, so everyone has freedom of what to use... Space or tabs.

File: package.json (hunk 1 above)
Comment (Caleb335) [reply package.json:23]: Alright I'd add this also

File: .prettierrc (hunk 3 above)
[the remark this answers is not in the document]
Comment (madhuni) [reply .prettierrc:3]: I agree with the freedom but the standard on the repo should be maintained and spaces with width `2` are more readable compare to tabs of width `4` IMO.

File: .prettierrc (hunk 2 above)
Comment (madhuni) [reply .prettierrc]: I think you will need to run the prettier again so that this change is applied.

Comment (Caleb335) [reply .prettierrc]: My bad, forgot to do that

File: .prettierrc (hunk 3 above)
[the remark this answers is not in the document]
Comment (Caleb335) [reply .prettierrc:3]: Noted

**Comment by NehemiahK** [comment]
@Caleb335 is this ready to be merged?

**Comment by Caleb335** [comment]
Yes  @NehemiahK

**Pull request merged**